hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancer.java

```diff
@@ -63,16 +68,14 @@ public ContainerBalancer(
 
   /**
    * Start ContainerBalancer. Current implementation is incomplete.
-   *
-   * @param balancerConfiguration Configuration values.
    */
-  public void start(ContainerBalancerConfiguration balancerConfiguration) {
+  public void start() {
     this.balancerRunning = true;
 
     ozoneConfiguration = new OzoneConfiguration();
 
     // initialise configs
-    this.config = balancerConfiguration;
+    //this.config = balancerConfiguration;
```

Comment:
`ContainerBalancerConfiguration` can be updated with any CLI params and then used as a single source of configurations wherever needed.

Reply:
@siddhantsangwan Thanks for the review!  we can just specify some important parameters for now I think and then keep on adding params as required . we can add more params as we move forward with the implementation.

Reply:
Since these parameters are required in multiple classes, we can set these fields in the `ContainerBalancerConfiguration` class and then pass this object to wherever parameters are required. This way, all parameters will be present in one place instead of being get/set in different places. It could be better to maintain all the parameters in a single class. What do you think?